taichi/codegen/amdgpu/codegen_amdgpu.h

```diff
@@ -11,8 +11,9 @@ class KernelCodeGenAMDGPU : public KernelCodeGen {
  public:
   KernelCodeGenAMDGPU(const CompileConfig &config,
                       const Kernel *kernel,
+                      IRNode *ir,
```

Comment:
Do we plan to remove `kernel` from the `KernelCodeGen`? Or do we plan to remove `ir` from `Kernel`?